### PR TITLE
[Commands] Cleanup #spawnstatus Command.

### PR DESCRIPTION
**Pull request by Kinglykrab**
- Cleanup messages and logic.

### CODE REVIEW

File: zone/gm_commands/spawnstatus.cpp [hunk 1]
```diff
@@ -2,27 +2,41 @@
 
 void command_spawnstatus(Client *c, const Seperator *sep)
 {
-	if ((sep->arg[1][0] == 'e') | (sep->arg[1][0] == 'E')) {
-		// show only enabled spawns
-		zone->ShowEnabledSpawnStatus(c);
```
Comment (Akkadius) [inline zone/gm_commands/spawnstatus.cpp:7]: If the goal is to truly clean code up. All of these methods that are referenced as `Zone` scoped methods are only referenced by this GM command file. They could be inlined within the GM command file and removed from Zone.

* ShowEnabledSpawnStatus
* ShowDisabledSpawnStatus
* SpawnStatus
* ShowSpawnStatusByID

This should reduce code bloat in Zone and keep it specific to this command

File: zone/zone.cpp [hunk 2]
```diff
@@ -2043,100 +2043,168 @@ bool ZoneDatabase::LoadStaticZonePoints(LinkedList<ZonePoint *> *zone_point_list
...
+				"{} Spawn{} listed.",
+				spawn_count,
+				spawn_count != 1 ? "s" : ""
+			).c_str()
+		);
+	} else {
+		client->Message(
+			Chat::White,
+			fmt::format(
+				"{} Of {} spawn{} listed.",
```
Comment (Akkadius) [inline zone/zone.cpp:2133]: Of intentionally capitalized? vs `of`

File: zone/zone.cpp [hunk 3]
```diff
@@ -2043,100 +2043,168 @@ bool ZoneDatabase::LoadStaticZonePoints(LinkedList<ZonePoint *> *zone_point_list
...
+			).c_str()
+		);
+		return;
+	}
+	
+	if (status_type == "All") {
+		client->Message(
+			Chat::White,
+			fmt::format(
+				"{} Spawn{} listed.",
```
Comment (Akkadius) [inline zone/zone.cpp:2124]: Lowercase `spawn(s)` ?

File: zone/zone.cpp [hunk 4]
```diff
@@ -2043,100 +2043,168 @@ bool ZoneDatabase::LoadStaticZonePoints(LinkedList<ZonePoint *> *zone_point_list
...
-		if (iterator.GetData()->GetID() == spawnid)
-		{
-			if (iterator.GetData()->timer.GetRemainingTime() == 0xFFFFFFFF)
-				client->Message(Chat::White, "  %d: %1.1f, %1.1f, %1.1f: disabled", iterator.GetData()->GetID(), iterator.GetData()->GetX(), iterator.GetData()->GetY(), iterator.GetData()->GetZ());
-			else
-				client->Message(Chat::White, "  %d: %1.1f, %1.1f, %1.1f: %1.2f", iterator.GetData()->GetID(), iterator.GetData()->GetX(), iterator.GetData()->GetY(), iterator.GetData()->GetZ(), (float)iterator.GetData()->timer.GetRemainingTime() / 1000);
-
-			iSpawnIDCount++;
+	while (iterator.MoreElements()) {
+		if (iterator.GetData()->GetID() == spawn_id) {
```
Comment (Akkadius) [inline zone/zone.cpp:2151]: Could shorten things up a bit by putting just below the while

```cpp
auto &e = iterator.GetData();
```

Then `e->GetID()` etc.

Comment (Kinglykrab) [reply zone/zone.cpp:2151]: Done.

File: zone/zone.cpp (hunk 2 above)
Comment (Kinglykrab) [reply zone/zone.cpp:2133]: Will lowercase.

File: zone/zone.cpp (hunk 3 above)
Comment (Kinglykrab) [reply zone/zone.cpp:2124]: Will lowercase.

File: zone/gm_commands/spawnstatus.cpp (hunk 1 above)
Comment (Kinglykrab) [reply zone/gm_commands/spawnstatus.cpp:7]: Will consolidate them all in to the GM file and remove all the methods from zone.cpp/zone.h. Will just use || inside the loop to determine how to handle each type.

**Review by Akkadius — APPROVED** [review]

**Pull request merged**